File: app/models/persistence/user_profile.rb
class UserProfile
  include DataMapper::Resource

  property :id, Serial
  property :gender, Enum[:m, :f, :u], :required => true
  property :birthday, DateTime, :required => true
  property :zipcode, String, :default => ""
  property :created_ts, DateTime, :required => true
  property :update_ts, DateTime, :required => true
  property :deleted_ts, ParanoidDateTime
  #property :deleted, ParanoidBoolean, :default => false

  belongs_to :user
end

File: public/javascripts/mobile/app/model/UserProfile.js
Ext.define('Genesis.model.UserProfile',
{
   extend : 'Ext.data.Model',
   alternateClassName : 'UserProfile',
   id : 'UserProfile',
   config :
   {
      belongsTo : [
      {
         model : 'Genesis.model.User',
         getterName : 'getUser',
         setterName : 'setUser'
      }],
      fields : ['gender', 'birthday', 'zipcode', 'created_ts', 'update_ts', 'user_id']
   },
   getUser : function()
   {

   }
});
